tests/benchq/algorithms/test_ld_gsee.py

```diff
@@ -0,0 +1,44 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+import numpy as np
+import pytest
+
+from benchq.algorithms.ld_gsee import (
+    get_ld_gsee_max_evolution_time,
+    get_ld_gsee_num_circuit_repetitions,
+)
+
+
+@pytest.mark.parametrize(
+    "alpha,energy_gap,square_overlap,precision,expected_result",
+    [
+        (1, 1, 1, 1, 5 / np.pi * np.sqrt(2 * np.log(16 * np.sqrt(2 / np.pi)))),
+        (1, 10, 1, 10, 1 / 2 / np.pi * np.sqrt(2 * np.log(16 * np.sqrt(2 / np.pi)))),
+    ],
+)
+def test_get_ld_gsee_num_iterations(
+    alpha, energy_gap, square_overlap, precision, expected_result
+):
+    assert np.isclose(
```

Comment:
**note**: it might not be very important for this case, but for future uses consider using functions from `np.testing` instead of plain asserts when comparing numerical values.

Functions from `numpy.testing` provide additional feedback like max relative/absolute differences - especially useful if you were comparing arrays of values.

Reply:
Good to know! Fixed in a204bb54cb94721d86a00c206d182ab5f2b2264c